main.go

```diff
@@ -121,18 +121,18 @@ func command() func() {
 		return func() {
 			commands.AddService(config, service, *configFile)
 		}
-	case "remove":
-		if len(os.Args) <= 2 {
-			return nil
+	case "-v":
+		version, err := ioutil.ReadFile(".version")
```

Comment:
Unfortunately, It won't work cause the `.version` file is not delivered within the binary. Plus we already have an option for showing version ([here](https://github.com/cristianoliveira/ergo/blob/master/main.go#L147)) which is working as expected.

The problem here is when installing ergo by `go install` or `go get` since the VERSION is not filled cause it gets code from the `master` branch and the `.version` inside it doesn't actually reflect the version on this branch. 

Please follow the issue thread for more details. Probably we will decide for filling `VERSION` variable with some special value for those cases.